src/promptflow-devkit/promptflow/_sdk/_orm/run_info.py

```diff
@@ -49,10 +49,11 @@ class RunInfo(Base):
     end_time = Column(TEXT)  # ISO8601("YYYY-MM-DD HH:MM:SS.SSS"), string
     data = Column(TEXT)  # local path of original run data, string
     run_source = Column(TEXT)  # run source, string
+    init_kwargs = Column(TEXT)  # init kwargs for flex flow
```

Comment:
@zhengfeiwang  shall we store this in properties?

Reply:
oh, if possible, i think it will be better to put it there.

Reply:
Moved under properties